backend/modules/metadata_store/prismastore.py

```diff
@@ -447,7 +447,7 @@ async def aget_data_ingestion_runs(
         """Get all data ingestion runs for a collection"""
         try:
             data_ingestion_runs = await self.db.ingestionruns.find_many(
-                where={"collection_name": collection_name}
+                where={"collection_name": collection_name}, order={"id": "desc"}
```

Comment:
Should also add this Collection listing APIs?

Reply:
Sure, will do. So that newer collections are on the top.